cron/update_lunch.py

```diff
@@ -1,31 +1,38 @@
 import datetime
 import logging
+import os
+
+from google.cloud import ndb
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "service_account.json"
+client = ndb.Client()
 
 import requests
 
 # Globals
 TIME_FORMAT = "%Y%m%dT%H%M%S"
-LUNCH_URL = (
-    "http://www.eastsideprep.org"
-    + "/wp-content/plugins/dpProEventCalendar/includes/ical.php?calendar_id=19"
-)
+LUNCH_URL = "http://www.eastsideprep.org/wp-content/plugins/dpProEventCalendar/includes/ical.php?calendar_id=19"
 
 # NDB class definitions
 
-"""
+
 class Lunch(ndb.Model):
     summary = ndb.StringProperty(required=True)
     # description is a list of lines in the description
     description = ndb.StringProperty(repeated=True)
     day = ndb.DateProperty(required=True)
 
+    @classmethod
+    def queryWithTimeConstraint(cls, earliest_lunch):

```

Comment:
The correct python code style would be `query_with_time_constraint`

Reply:
fixed